src/steadystate.jl

```diff
@@ -106,11 +106,11 @@ function _steadystate(
     idx_range = collect(1:N)
     rows = _get_dense_similar(L_tmp, N)
     cols = _get_dense_similar(L_tmp, N)
-    datas = _get_dense_similar(L_tmp, N)
+    _data = _get_dense_similar(L_tmp, N)
```

Comment:
I think it is better to leave `datas` and add it to the words to exclude. We should keep the code readable

Reply:
but `data` is an uncountable noun @@

Reply:
or something like `L_tmp2` ?

Reply:
This is just a temporary variable
So I prefer not adding it to the exclude list

Reply:
Ok, maybe `nzvals` or `vals`. What I don't like is that is is the only variable written with that underscore.

Reply:
ok, changed to `vals`